dataframe/dataframe.go

```diff
@@ -887,15 +898,23 @@ func LoadRecords(records [][]string, options ...LoadOption) DataFrame {
 	if cfg.hasHeader && len(records) <= 1 {
 		return DataFrame{Err: fmt.Errorf("load records: empty DataFrame")}
 	}
+	if len(cfg.names) > len(records[0]) {
```

Comment:
> Also application of this option should only happen if len(cfg.names) != 0

A range over something with a length of 0 will not loop at all. So it's fine to not explicitly check for that.